docker/docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: "3.7"
+services:
+  app:
+    container_name: autogpt-gotty
+    image: cdukes/autogpt-gotty:latest
```

Comment:
Cool Idea, Is Dockerfile open-sourced? Don't see any reference in at hub https://hub.docker.com/r/cdukes/autogpt-gotty or can we have it Commited here, so anyone could build it?

Reply:
Yes, are you unfamiliar with Docker?

Reply:
>or can we have it Commited here, so anyone could build it?

You can, but don't need to. Any future changes would just get built in. Each time it is built, it will pull the Auto-GPT repo and build automatically. I did document the Dockerfile contents in the readme just in case someone wants to tinker, but it's also [available on dockerhub](https://hub.docker.com/layers/cdukes/autogpt-gotty/latest/images/sha256-c07d1d389552946308e132c88cca906ccb56eec414987ede75306e28d38ef345?context=explore)